wsmaster/codenvy-hosted-license-shared/src/main/java/com/codenvy/license/shared/InvalidLicenseException.java

```diff
@@ -12,7 +12,7 @@
  * is strictly forbidden unless prior written permission is obtained
  * from Codenvy S.A..
  */
-package com.codenvy.api.license;
+package com.codenvy.license.shared;
```

Comment:
It looks strange to me to locate exception in shared module. Can you use it by client's modules?